farm/modeling/prediction_head.py

```diff
@@ -258,16 +241,32 @@ class TextClassificationHead(PredictionHead):
     def __init__(
         self,
         layer_dims=[768,1],
```

Comment:
I think having these defaults can now become confusing (e.g. people not supplying any of layer_dims and num_labels). 
How about:
- layer_dims = None 
- in init: 
``` 
if num_labels: 
     self.layer_dims = [768, num_labels]   
elif layer_dims: 
     self.layer_dims = layer_dims                
      (and maybe raise a deprecation warning)
else: 
    raise ValueError("Please supply `num_labels` to define output dim of prediction head" )
```

Reply:
Yup agreed. This is fixed in the latest commit.